dev/tools/make_git_revision.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
Is bash absolutely required here? I think we have tried until then to keep the main Makefile shell agnostic (this doesn't apply to e.g. the CI or test-suite Makefile AFAIR).

Reply:
> Is bash absolutely required here? I think we have tried until then to keep the main Makefile shell agnostic (this doesn't apply to e.g. the CI or test-suite Makefile AFAIR).

The build system is far from agnostic to `bash`.

Reply:
As long as other build system maintainers are happy with this, I don't mind.